website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -112,6 +112,65 @@ For Slack and Microsoft Teams notifications, the payload conforms to the respect
 }
 ```
 
+### Workspace Notification Payload
+
+| Name                                         | Type   | Description                                                               |
+| -------------------------------------------- | ------ | ------------------------------------------------------------------------- |
+| `payload_version`                            | number | Currently always "2".                                                     |
+| `notification_configuration_id`              | string | ID of the configuration which defined this notification.                  |
+| `notification_configuration_url`             | string | URL of the Notification Configuration.                                    |
+| `trigger_scope`                              | string | "assessment" for workspace notifications.                                 |
+| `trigger`                                    | string | Value of the trigger which caused the notification to be sent.            |
+| `message`                                    | string | Human-readable reason for the notification.                               |
+| `details`                                    | object | Object containing details specific to the notification.                   |
+| `details.new_assessment_result`              | object | The most recent Assessment Result which triggered the notification.       |
+| `details.new_assessment_result.id`           | string | ID of the Assessment Result.                                              |
```

Comment:
Let's not capitalize assessment result. Do we capitalize things like this in other places? In the regular docs, it's best practice that we do not capitalize feature names, etc. But lmk if I'm incorrect here about how we typically do API

Reply:
Nope, that was unintentional.